Examples/SimpleMacApp/SimpleMacApp/CKMSampleTableComponent.h

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <ComponentKit/ComponentKit.h>
+
+/**
+ * This is just a sample of how you can insert a tableview as a component into a bigger layout.
+ * It makes an NSTableView with one column.
+ */
+@interface CKMSampleTableComponent : CKComponent
```

Comment:
Hm, usually you want to layout components within table view. But in this case you are laying out table view as component. Why? Is it the case when you want to built your app entirely on top of components?